src/app/cluster/list/template.html

```diff
@@ -17,7 +17,7 @@
 <ng-container [ngSwitch]="areExternalClustersEnabled">
   <mat-tab-group *ngSwitchCase="true"
                  animationDuration="0ms"
-                 [selectedIndex]="clusterIndex">
+                 [selectedIndex]="clusterIndex? 1 : 0">
```

Comment:
This is the part that we can still improve I think. At the moment there is no difference what will be the value of fragment, if it is not null then the tab will be switched.